bin/coverage_doctest.py

```diff
@@ -569,7 +569,7 @@ def go(sympy_top, file, verbose=False, no_color=False, exact=True, sphinx=True):
             total_sphinx += _total_sphinx
             num_functions += _num_functions
         return doctests, total_sphinx, num_functions
-    if (not (file.endswith('.py') or file.endswith('.pyx')) or
+    if (not (file.endswith((".py", ".pyx"))) or
```

Comment:
I didn't know it was possible to do this.

Reply:
Me, neither!